webrtc.js

```diff
@@ -61,9 +61,11 @@ var respecConfig = {
     { name: "Bernard Aboba", company: "Microsoft Corporation", note: "until March 2017",
       w3cid: "65611"
     },
-    { name: "Taylor Brandstetter", company: "Google",
+    { name: "Taylor Brandstetter", company: "Google", note: "until June 2018",
       w3cid: "82908"
     },
+    { name: "Henrik Boström", company: "Google",
+    },
```

Comment:
Is there a specific order to this list (I just placed my name next to the other Googler)? And how do I know my w3cid?

Reply:
@henbos You can go to www.w3.org/users/myprofile and id will be on the right (and in the uri, even). Manual of style says nothing about order: https://w3c.github.io/manual-of-style/#Editors

Reply:
Can you add the w3cid?

Reply:
Thanks, done!